notifiers/notifiers.go

```diff
@@ -14,10 +14,10 @@ type SlackSpecific struct {
 }
 
 type Notification struct {
-	Title   string                         `json:"title,omitempty"`
-	Body    string                         `json:"body,omitempty"`
-	Slack   *SlackNotification             `json:"slack,omitempty"`
-	Webhook map[string]WebhookNotification `json:"webhook,omitempty" patchStrategy:"replace"`
+	Title   string                         `json:"title,omitempty" yaml:"title,omitempty"`
```

Comment:
Is `yaml` really required? The `github.com/ghodss/yaml` re-uses `json` annotation to marshal/unmarshal YAML. I think we should keep only json annotations to avoid confusion.